at_contacts_group_flutter/example/pubspec.yaml

```diff
@@ -61,6 +58,10 @@ dev_dependencies:
 # For information on the generic Dart part of this file, see the
 # following page: https://dart.dev/tools/pub/pubspec
 
+dependency_overrides:
+  biometric_storage: ^4.1.3
+  at_onboarding_flutter: ^5.0.3
+  at_backupkey_flutter: ^4.0.6
```

Comment:
@Pritam-deb can we remove onboarding and backupkey from overrides and move dependencies?

Reply:
moving them from overrides to dependencies doesn't work.